11.0.Dockerfile

```diff
@@ -221,6 +221,7 @@ ONBUILD RUN mkdir -p /opt/odoo/custom/ssh \
             && ln -s /opt/odoo/custom/ssh ~root/.ssh \
             && chmod -R u=rwX,go= /opt/odoo/custom/ssh \
             && sync
+ONBUILD ARG DB_VERSION=None
```

Comment:
This is a bit misleading. I think the best is to make it default to "latest" instead. You'll have to fix the other dockerfiles and the build script too.